sidecar/sokuji_sidecar/gpt_sovits/g2p/english/EnglishG2P.py

```
@@ -0,0 +1,304 @@
+CMU_DICT_HOT_PATH = os.path.join(assets.english_g2p_dir(), "engdict-hot.rep")
+# SOKUJI: upstream ships these caches as pickles; unpickling files from a
+# downloaded (and env-overridable) model repository is arbitrary code
+# execution by design, so the Sokuji model card publishes them as JSON
+# (same dict[str, list] payload) and this port loads JSON only.
+CACHE_PATH = os.path.join(assets.english_g2p_dir(), "engdict_cache.json")
+NAMECACHE_PATH = os.path.join(assets.english_g2p_dir(), "namedict_cache.json")
+MODEL_PATH = os.path.join(assets.english_g2p_dir(), "checkpoint20.npz")
+
+# 正则表达式和映射
+REP_MAP = {
+    "[;:：，；]": ",",
```

Ruff (0.15.21)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)